database/src/main/java/dev/msfjarvis/lobsters/model/SubmitterAdapter.kt

```diff
@@ -0,0 +1,20 @@
+package dev.msfjarvis.lobsters.model
+
+import com.squareup.moshi.Moshi
+import com.squareup.moshi.adapter
+import com.squareup.sqldelight.ColumnAdapter
+import dev.zacsweers.moshix.reflect.MetadataKotlinJsonAdapterFactory
+
+@OptIn(ExperimentalStdlibApi::class)
+class SubmitterAdapter : ColumnAdapter<Submitter, String> {
+  private val moshi = Moshi.Builder().add(MetadataKotlinJsonAdapterFactory()).build()
```

Comment:
Should probably have these adapters be provided by Dagger so we can make the Moshi initialization centralised as well

Reply:
Will do it later after fixing the invalidation issue